Sources/LiCore/QueryUtil/LiQuerySetting.swift

```diff
@@ -96,6 +98,11 @@ public class LiQuerySetting {
         } else {
             self.limit = nil
         }
+        if let offset = data["offset"] as? String {
+            self.offset = offset
+        } else {
+            self.offset = nil
+        }
```

Comment:
Couldn't you write this as ..
`self.offset = data["offset"] as? String`

Reply:
Will update this